triveni-codev/GBWebApp-dev
# Darlp networks delete channel 1204317704200090

Hi @triveni-codev ,

Kindly have some to check this PR. This is a branch out from PR585 which was already approved.

ASANA: https://app.asana.com/0/1201943683850147/1204317704200090

To Test:
-single and multiple delete feature on Channel using  the recent multiple selection UI
-edit a Channel using the recent multiple selection UI
-add a Channel using  the recent multiple selection UI
-you may test either from server or newly created/uncommitted Channel
-big commit is already implemented on delete feature
-new UI includes vertical buttons on the right side
-edit button automatically enables once a Channel is selected and disables on multiple selected Channel or no Channel selected
-delete button automatically enables on one or more Channels and disables if no Channel is selected
-switching between tabs should automatically RESET the selection and disables both Delete and Edit buttons
-after SUCCESSFULLY Adding, Deleting and Updating a Channel, it should RESET the selection and disables both Delete and Edit buttons


Expected errors and DON'T TEST yet since I have to coordinate with @chriscodev for possible fixes
-after cancelling a delete confirmation box, multiple selection gets buggy (kindly refresh for now)
-after doing big commit, multiple selection gets buggy (kindly refresh for now)
